src/app/shell/personal-cabinet/shared-cabinet/messages/messages.component.ts

```diff
@@ -98,15 +96,9 @@ export class MessagesComponent extends CabinetDataComponent {
   getProviderWorkshops(): void {
     if (this.subRole === Role.None) {
       this.store.dispatch(new GetWorkshopListByProviderId(this.providerId));
-    } else {
-      this.store.dispatch(new GetProviderAdminWorkshops());
     }
   }
 
-  getChats(): void {
-    this.store.dispatch(new GetUserChatRooms(this.chatRoomsParameters));
-  }
-
   setListeners(): void {
     this.chatRooms$
       .pipe(filter(Boolean), takeUntil(this.destroy$))
```

Comment:
I belive that for search value change you should set the first page, correct?
because entering new text will give you no result on the second page

Reply:
![image](https://user-images.githubusercontent.com/27493959/215972219-dec0b3d3-613a-428f-8dc3-70a56c11115f.png)
![image](https://user-images.githubusercontent.com/27493959/215972184-abb4ce79-2258-4b98-b138-7b5d5544eee3.png)